packages/fx-core/resource/package.nls.json

```diff
@@ -719,13 +719,17 @@
   "core.SampleSelect.buttons.viewSamples": "View samples",
   "core.HostTypeOptionAzure.label": "Azure",
   "core.HostTypeOptionSPFx.label": "SharePoint Framework (SPFx)",
-  "core.updateBotIdsQuestion.title": "We will create a new bot for you to get started with Teams Toolkit",
+  "core.updateBotIdsQuestion.title": "Create new bot(s) for debugging",
   "core.updateBotIdsQuestion.placeholder": "Unselect to keep with the original one.",
-  "core.updateContentUrlQuestion.title": "We will update the content url of following tab(s) for you to get started with Teams Toolkit",
-  "core.updateWebsiteUrlQuestion.title": "We will update the website url of following tab(s) for you to get started with Teams Toolkit",
-  "core.updateContentUrlOption.description": "update the content url from %s to %s",
-  "core.updateWebsiteUrlOption.description": "update the website url from %s to %s",
-  "core.updateUrlQuestion.placeholder": "Unselect to keep with the original url",
+  "core.updateBotIdForBot.description": "update botId %s to \"{{BOT_ID}}\" in manifest.json",
```

Comment:
to be consistent, to capitalize the first character: u -> U

Reply:
updated strings. Please see the attachments above :)